### PR TITLE
Update PCA_analysis.qmd and added 3d plots

### PR DESCRIPTION
**Pull request by samrit2442**
Added 3d visualisation of PCA.
Please review @DrLynTaylor @statasaurus

### CODE REVIEW

File: R/PCA_analysis.qmd [hunk 1]
```diff
@@ -55,3 +56,41 @@ plt
...
+## Visualization of PCA in 3d Scatter Plot
+
+A 3d scatter plot allows us to see the relationships between three principle components simultaneously and also gives us a better understanding of how much variance is explained by these components.
+
+It also allows for interactive exploration where we can rotate the plot and view it from a different angles.
+
+We will plot this using `plotly` package.
+
+```{r}
+library(plotly)
```
Comment (statasaurus) [inline R/PCA_analysis.qmd:69]:
```suggestion

```
You are calling plotly twice in the script it is unnecessary

**Review by statasaurus — CHANGES_REQUESTED** [review]
Just a minor change but it looks good

**Comment by samrit2442** [comment]
Yes. I approved the changes.

**Review by samrit2442 — COMMENTED** [review]
Made the changes.

**Review by statasaurus — APPROVED** [review]
Looks great!

**Pull request merged**